### PR TITLE
CHANGELOG-2.3.md: content for v2.3.0

### PR DESCRIPTION
**Pull request by pohly**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

Once this is merged, we can release v2.3.0.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by pohly** [comment]
/assign @msau42 

Can you also do the release?

File: CHANGELOG-2.3.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Changelog since v2.2.0
+
+## New Features
+
+- Add '--csi.testvolumeexpandsize` flag ([#219](https://github.com/kubernetes-csi/csi-test/pull/219), [@wnxn](https://github.com/wnxn))
+
+
+## Bug Fixes
```
Comment (msau42) [inline CHANGELOG-2.3.md]: I think https://github.com/kubernetes-csi/csi-test/pull/217 is worthy of a release note. I updated the PR with one. Could you add it here?

Comment (pohly) [reply CHANGELOG-2.3.md]: Done. While looking at the changelog again I noticed that capitalization and past vs. present tense was inconsistent; it's a bit better now.

Comment (msau42) [reply CHANGELOG-2.3.md]: still don't see it?

Comment (pohly) [reply CHANGELOG-2.3.md]: Darn, missing `-a` in `git commit --amend`. Done now.

**Pull request merged**